tests/functionaltests/UXTestHelpers.h

```diff
@@ -81,4 +84,22 @@ class XamlEventSubscription {
     StrongId<WXDependencyProperty> _propertyToObserve;
 }; // class XamlEventSubscription
 
+// Class used to signal and wait during a test run
+class UXEvent {
+public:
+    UXEvent() : _signaled(false) {
+        _condition.attach([[NSCondition alloc] init]);
+    }
+
+    UXEvent(const UXEvent& other) = delete; // no copy
+    UXEvent& operator=(const UXEvent& other) = delete;
+
+    void Signal();
```

Comment:
nit: I'd probably call this 'Set' to align with 'Reset'

---
In reply to: [98240403](https://github.com/Microsoft/WinObjC/pull/1855#discussion_r98240403) [](ancestors = 98240403)